### PR TITLE
Add option to ignore unknown subbed releases

### PR DESCRIPTION
**Pull request by fernandog**
@medariox @duramato

### CODE REVIEW

File: sickbeard/show_name_helpers.py [hunk 1]
```diff
@@ -31,13 +31,15 @@
 from collections import namedtuple
 
 resultFilters = [
-    "sub(bed|ed|pack|s)",
     "(dir|sub|nfo)fix",
     "(?<!shomin.)sample",
     "(dvd)?extras",
     "dub(bed)?"
```
Comment (duramato) [inline sickbeard/show_name_helpers.py]: Add this one too

Comment (fernandog) [inline sickbeard/show_name_helpers.py]: not the scope of this PR

Comment (duramato) [inline sickbeard/show_name_helpers.py]: It is, those are dubbed

Comment (fernandog) [inline sickbeard/show_name_helpers.py]: "Add option to ignore unknown subbed releases" - SUBBED

Comment (duramato) [inline sickbeard/show_name_helpers.py]: Then add it too its as simple as changing the setting text, becouse as is you are ignoring scene releases without an option.
This is what was talked on irc @medariox

**Comment by fernandog** [comment]
@medariox waiting you test and/or approve

**Comment by medariox** [comment]
Approved

Thanks!

**Pull request merged**